studio/src/rte/ost.js

```diff
@@ -86,6 +92,7 @@ const OST_OPTION_ATTRIBUTE_MAPPING = {
     displayTax: 'data-display-tax',
     forceTaxExclusive: 'data-tax-exclusive',
     isPerpetual: 'data-perpetual',
+    storedPromoOverride: 'data-promotion-code',
```

Comment:
this will be fixed in https://github.com/adobecom/mas/pull/118/files#diff-797a3a33d7fd7d93374be8a2fe79e6fe3aab46f3c6a7a0db7f56553556d2b01eR92
i think you can remove it unless its needed to verify the pr

Reply:
Thanks, removed.